### ISSUE TITLE
Point3D Offsets do not offset correctly by Y and Z

### ISSUE THREAD

**Issue by sschr15** [DonBruce64/MinecraftTransportSimulator#241]
I have not actually played with this, but as I was looking through the source, I saw that the offset function actually offsets all the directions by the X offset. I'd just like to let you know while I'm looking here.
```java
public void offset(double xOffset, double yOffset, double zOffset){
	this.x += xOffset;
	this.y += xOffset;
	this.z += xOffset;
}
```

**Comment by DonBruce64** [comment]
Dang, nobody has ever gotten that deep into the code.  That class was one I made in anticipation in moving away from Minecraft's Vec3D class, and isn't currently used anywhere, so I probably should just delete it.  Will fix.

**Comment by DonBruce64** [comment]
Corrected in V15.9.0.
